Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:latest
+
+LABEL maintainer="haisam.ido@gmail.com"
+
+RUN mkdir openmct
+WORKDIR /openmct
+
+# Mounting a volume would be better
+ADD . .
+
+RUN rm -rf dist
+RUN rm -rf node_modules
```

Comment:
This looks like a bad approach, you already created large layers potentially including those files, that just inflates whole build process unnecessarily by copying node_modules into container in the first place. You should either copy wanted dirs only or use .dockerignore file to blacklist those.